src/encoded/static/scss/encoded/modules/_forms.scss

```diff
@@ -105,6 +113,18 @@ input[type="text"], input[type="password"], input[type="date"], input[type="date
 	}
 }
 
+.navbar-search-form {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+}
+
+.search-button {
+	background-color: white;
+    border: 0 none white; 
```

Comment:
You could just use `border: none` here. (Or `border: 0`)

Reply:
Good point. Corrected.